packages/cardano-services/src/NetworkInfo/DbSyncNetworkInfoProvider/mappers.ts

```diff
@@ -47,9 +49,11 @@ export const toNetworkInfo = ({
 export const loadGenesisData = async (cardanoNodeConfigPath: string): Promise<GenesisData> => {
   try {
     const genesisFilePath = require(path.resolve(cardanoNodeConfigPath)).ShelleyGenesisFile;
-    const genesis = require(path.resolve(path.dirname(cardanoNodeConfigPath), genesisFilePath));
+    const genesis = JSONbig.parse(
+      fs.readFileSync(path.resolve(path.dirname(cardanoNodeConfigPath), genesisFilePath), 'utf-8')
+    );
     return {
-      maxLovelaceSupply: genesis.maxLovelaceSupply,
+      maxLovelaceSupply: genesis.maxLovelaceSupply.toString(),
```

Comment:
I'm expecting this to be a `bigint`, which is the desired type:

https://github.com/input-output-hk/cardano-js-sdk/blob/a91088e969a2ce3f6471a12644bb6dc45d4813e4/packages/cardano-services/src/NetworkInfo/DbSyncNetworkInfoProvider/types.ts#L22

https://github.com/input-output-hk/cardano-js-sdk/blob/a91088e969a2ce3f6471a12644bb6dc45d4813e4/packages/core/src/Cardano/types/Value.ts#L4

Reply:
5dd83f2c65971592dd40966b7a5036ecf26fa512